index.bs

```diff
@@ -4341,6 +4365,12 @@ lists and names some [=authenticator types=] of particular interest.
                 <td> [=client-side credential storage modality|Client-side storage=] </td>
                 <td> [=Multi-factor capable=] </td>
             </tr>
+            <tr>
+                <th> <dfn>Passkey platform authenticator</dfn> </th>
```

Comment:
If we're introducing the "passkey" term, should we also rename "first-factor roaming authenticator" to "passkey roaming authenticator"? The "first-factor" term has never really caught on and is awkwardly mismatched with "user-verifying platform authenticator", and there's only one use of it outside the Authenticator Taxonomy section.

Reply:
On a similar train of thought: should we remove the "User-verifying platform authenticator" term from this list? It's largely superseded by the passkey class. We could demote it to the below list of less interesting combinations, since it's unlikely that a platform authenticator would be multi-factor capable but not client-side-storage capable.

Reply:
What is the `passkey platform authenticator`? We have passkey, platform authenticator and cross-platform authenticator. For me, the term is quite vague. Is it better to introduce syncable, multi-device or something?
And we have some of 3rd party passkey providers as well (which is not the platform authenticator). I think it's time to re-define the authenticator taxonomy.

Reply:
> And we have some of 3rd party passkey providers as well (which is not the platform authenticator). 

3P passkey providers are considered the same as 1P passkey providers and can be both platform and cross-platform, which is what this change reflects.

Reply:
Resolving the comment as there's been no activity in over a month